docs/installation/google.md

```diff
@@ -41,7 +41,7 @@ parent = "smn_cloud"
 Read more about [deploying Containers on Google Cloud Platform][5].
 
 [1]: https://cloud.google.com/console
```

Comment:
This redirects me to https://console.developers.google.com/project  not sure what the canonical URL is?